test/aggregations/terms-aggregation.js

```diff
@@ -14,4 +15,21 @@ describe('termsAggregation', () => {
     })
   })
 
+    it('should create a terms aggregation with custom options', () => {
+      let options = {
+        order: {
+          timestamp: 'asc'
+        },
+        size: 0
+      };
+      let result = termsAggregation('user', null, options)
+      expect(result).to.eql({
+        agg_terms_user: {
+          terms: _.extend({
```

Comment:
Nitpicking: Let's keep unit tests as dumb as possible, so instead of using lodash extend just assert with the expected result.